tests/testthat/test-compiler-errors.R

```diff
@@ -195,5 +195,5 @@ test_that("animint(out.dir = 'dir1', out.dir = 'dir2') is an error", {
 test_that("animint() is an error", {
   expect_error({
     animint()
-  }, "No arguments passed to animint. Arguments should include ggplot(1 or more) and options(0 or more)")
```

Comment:
the old version is more correct english (although I would say ggplots instead of ggplot), can you please revert?
why did you make this change?
did you know about expect_error(something, "message", fixed=TRUE) ?

Reply:
Reverted the error message to the old one and used fixed=TRUE. I made this change because I thought it would be better to avoid '()' in the error message sentence. I had used escape qualifiers to do the pattern matching in the test and didn't know about fixed=TRUE !